coalib/settings/Setting.py

```diff
@@ -13,6 +14,22 @@ def path_list(obj, *args, **kwargs):
     return obj.__path_list__(*args, **kwargs)
 
 
+def glob(obj, *args, **kwargs):
+    """
+    Creates a path in which all special glob characters in all the
+    parent directories in the given setting are properly escaped.
+    """
```

Comment:
I hadd a pass through to the function underneath because it had some other parameter concerning the absoluteness of the path or something IIRC.